release.config.js

```diff
@@ -4,15 +4,26 @@
  *
  * `semantic-release` runs with the following configuration via the `release.yml` github action.
  */
+const { checkVersion } = require("./scripts/manageDeprecations");
+
 const config = {
-  branches: ["main"],
+  branches: ["main", "release/NDS-1363-new-plugin"],
   plugins: [
     [
       "@semantic-release/commit-analyzer",
       {
         preset: "conventionalcommits",
       },
     ],
+    [
+      "semantic-release-npm-deprecate-old-versions",
+      {
+        rules: [
+          checkVersion, // our own custom rule, so we can manage deprecated version ranges
```

Comment:
according to docs, this _should_ be fine, but....

```
AggregateError: 
    TypeError: Cannot read properties of undefined (reading 'rule')
        at /home/runner/work/design_system/design_system/node_modules/semantic-release-npm-deprecate-old-versions/dist/configuration-loader.js:23:28
        ```
investigating the cause...